repos/system_upgrade/common/actors/enablerhsmtargetrepos/libraries/enablerhsmtargetrepos.py

```diff
@@ -17,13 +17,13 @@ def set_rhsm_release():
         rhsm.set_release(mounting.NotIsolatedActions(base_dir='/'), target_version)
     except CalledProcessError as err:
         api.current_logger().warning('Unable to set the {0} release through subscription-manager. When using dnf,'
-                                     ' content of the latest RHEL 8 minor version will be downloaded.\n{1}'
-                                     .format(target_version, str(err)))
+                                     ' content of the latest RHEL {1} minor version will be downloaded.\n{2}'
+                                     .format(target_version, target_version.split('.')[0], str(err)))
```

Comment:
as a follow up: This probably should you get_target_major_version instead

Reply:
good catch. agree that we should do it in a followup